test/controllers/update/overseas.entity.presenter.controller.spec.ts

```diff
@@ -75,7 +77,7 @@ describe("OVERSEAS ENTITY PRESENTER controller", () => {
       const resp = await request(app).get(OVERSEAS_ENTITY_PRESENTER_URL);
 
       expect(resp.status).toEqual(200);
-      expect(resp.text).toContain(OVERSEAS_ENTITY_PRESENTER_PAGE_TITLE);
+      expect(resp.text).not.toContain(OVERSEAS_ENTITY_PRESENTER_PAGE_TITLE);
```

Comment:
The existing presenter page tests to be left unchanged and the title on that page should still be the same (it's just slightly different if on the remove journey). Think this is related to you changing the `OVERSEAS_ENTITY_PRESENTER_PAGE_TITLE` value in the mocks.

Reply:
reverted